spec/tic_tac_toe_core/computer_player_spec.rb

```diff
@@ -1,19 +1,80 @@
 require 'spec_helper'
+require 'tic_tac_toe_core/board'
 require 'tic_tac_toe_core/computer_player'
 
 module TicTacToeCore
   describe ComputerPlayer do
-    let(:player) { ComputerPlayer.new('X') }
+    let(:x_player) { ComputerPlayer.new('X') }
+    let(:o_player) { ComputerPlayer.new('O') }
           
     it "makes a valid board move" do
       board = Board.create_empty
-      next_move = player.next_move(board)
+      next_move = x_player.next_move(board)
 
       expect(board.is_available?(next_move)).to be true
     end
 
     it "has a marker" do
-      expect(player.marker).to eq('X')
+      expect(x_player.marker).to eq('X')
```

Comment:
I would delete this test, it's just testing that the class under test stores and returns the mark, no behaviour there to test. This is basically testing getters / setters.